pkg/volume/glusterfs/glusterfs.go

```diff
@@ -436,6 +439,89 @@ func (d *glusterfsVolumeDeleter) GetPath() string {
 	return d.plugin.host.GetPodVolumeDir(d.glusterfsMounter.glusterfs.pod.UID, strings.EscapeQualifiedNameForDisk(name), d.glusterfsMounter.glusterfs.volName)
 }
 
+//
+// Traverse the PVs, fetching all the GIDs from those
+// in a given storage class, and mark them in the table.
+//
+func (p *glusterfsPlugin) collectGids(class *storage.StorageClass) error {
```

Comment:
no need to pass in the min/max:
when this is called, we have the full range:
it is only called when the gidTable is newly instantiated, i.e. before resetting the range to the SC values